ocs_ci/ocs/amq.py

```diff
@@ -639,6 +640,40 @@ def validate_amq_benchmark(self, result, amq_workload_yaml, benchmark_pod_name="
 
         return res_dict
 
+    def export_amq_output_to_gsheet(self, amq_output, sheet_name, sheet_index):
+        """
+        Collect amq data to google spreadsheet
+
+        Args:
+            amq_output (dict):  amq output in dict
+            sheet_name (str): Name of the sheet
+            sheet_index (int): Index of sheet
+
+        """
+        # Collect data and export to Google doc spreadsheet
+        g_sheet = GoogleSpreadSheetAPI(
+            sheet_name=sheet_name, sheet_index=sheet_index
+        )
+        log.info("Exporting amq data to google spreadsheet")
+
+        headers_to_key = []
+        values = []
+        for key, val in amq_output.items():
+            headers_to_key.append(key)
+            values.append(val)
+
+        # Update amq_result to gsheet
+        g_sheet.insert_row(values, 2)
+        g_sheet.insert_row(headers_to_key, 2)
+
+        # Capturing versions(OCP, OCS and Ceph) and test run name
+        g_sheet.insert_row(
+            [f"ocp_version:{utils.get_cluster_version()}",
+             f"ocs_build_number:{utils.get_ocs_build_number()}",
+             f"ceph_version:{utils.get_ceph_version()}",
+             f"test_run_name:{utils.get_testrun_name()}"], 2
+        )
+
```

Comment:
Please share the link to the spreadsheet generated. I would like see how contents are organized.

Reply:
https://docs.google.com/spreadsheets/d/112aodIzzBmScZBXqBNqq4TK974PtR5pQWXe93gzwgzg/edit#gid=1443956424